tests/src/main_tests.rs

```diff
@@ -219,15 +222,16 @@ async fn import_expected_notes() {
     let input_note = client_2.get_input_note(note.id()).await.unwrap().unwrap();
 
     // If imported before execution then the inclusion proof should be None
-    assert!(input_note.inclusion_proof().is_none());
+    assert!(input_note.inclusion_proof().is_none(), "Unexpectedly found inclusion proof");
+    assert!(matches!(input_note.state(), InputNoteState::Expected { .. }));
```

Comment:
I feel like the first check here is redundant. The second check should always mean that the inclusion proof is none. We could maybe change it to:

```suggestion
    // If imported before execution, the note should be imported as `Expected`
    assert!(matches!(input_note.state(), InputNoteState::Expected { .. }));
```